### PR TITLE
[inner-rce] save editorState instead of contentState

### PR DESCRIPTION
**Pull request by danarish4**

### CODE REVIEW

File: packages/plugin-accordion/web/src/components/domain/accordion.ts [hunk 1]
```diff
@@ -77,23 +76,23 @@ export class Accordion {
...
     this.updateData({ ...this.getData() });
   };
 
-  createNewPair = () => {
+  createNewPair = (): Pair => {
     return {
       key: generateKey(),
-      title: __convertToRawWithoutVersion(EditorState.createEmpty().getCurrentContent()),
-      content: __convertToRawWithoutVersion(EditorState.createEmpty().getCurrentContent()),
+      title: EditorState.createEmpty(),
```
Comment (shaulgo) [inline packages/plugin-accordion/web/src/components/domain/accordion.ts:94]:
```suggestion
      title: undefined,
```

File: packages/editor/web/src/RichContentEditor/InnerRCE.jsx [hunk 2]
```diff
@@ -41,20 +37,8 @@ class InnerRCE extends Component {
...
-    } else {
-      return null;
-    }
-  }
-
   saveInnerRCE = editorState => {
-    this.setState({ editorState });
-    const newContentState = __convertToRawWithoutVersion(editorState.getCurrentContent());
-    this.props.onChange(newContentState);
+    this.props.onChange(editorState);
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/InnerRCE.jsx:41]: remove function. Pass props.onChange directly

File: packages/editor/web/lib/editorStateConversion.ts [hunk 3]
```diff
@@ -41,23 +46,62 @@ const convertAnchorTypeForUnsupportedInOneApp = rowContentState => {
...
+type Pair = {
+  key: string;
+  title: EditorState;
+  content: EditorState;
+};
 
-const __convertToRawWithoutVersion = ContentState =>
-  fixBlockDataImmutableJS(convertAnchorTypeForUnsupportedInOneApp(toRaw(ContentState)));
+const convertInnerRceToRaw = rawContentState => {
+  const updatedRaw = cloneDeep(rawContentState);
```
Comment (shaulgo) [inline packages/editor/web/lib/editorStateConversion.ts:56]: cloneWithDeep uses customizer function. Check loadash docs.
customizer = (val) => {
	if (val === editorState) return val;
	else return undefined
}

File: packages/editor/web/src/RichContentEditor/InnerRCE.jsx (hunk 2 above)
Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/InnerRCE.jsx:41]: done.

File: packages/plugin-accordion/web/src/components/domain/accordion.ts (hunk 1 above)
Comment (maimonav) [reply packages/plugin-accordion/web/src/components/domain/accordion.ts]: done.

File: packages/editor/web/lib/editorStateConversion.ts (hunk 3 above)
Comment (maimonav) [reply packages/editor/web/lib/editorStateConversion.ts]: done.

**Review by shaulgo — APPROVED** [review]
add tests for accordion in editorStateConversion.spec

**Review by shaulgo — APPROVED** [review]

**Pull request merged**